metadata-io/src/main/java/com/linkedin/metadata/entity/cassandra/CassandraEntityService.java

```diff
@@ -331,6 +333,33 @@ protected List<Pair<String, UpdateAspectResult>> ingestAspectsToLocalDB(
     }, DEFAULT_MAX_TRANSACTION_RETRY);
   }
 
+  @Nullable
+  @Override
+  public RecordTemplate ingestAspectIfNotPresent(@Nonnull Urn urn, @Nonnull String aspectName,
+      @Nonnull RecordTemplate newValue, @Nonnull AuditStamp auditStamp, SystemMetadata systemMetadata) {
```

Comment:
nit: let's add annotation to system metadata for consistency

Reply:
@Nonnull ? I think?

Reply:
Yeah I think so, I'll add it onto ingestAspect too then, I copied the method definition from that 😆